pkg/commands/tls/custom/certificate/create.go

```diff
@@ -59,7 +59,11 @@ func (c *CreateCommand) constructInput() *fastly.CreateCustomTLSCertificateInput
 	if c.id != "" {
 		input.ID = c.id
 	}
-	input.CertBlob = c.certBlob
+
+	if c.certBlob.WasSet {
+		input.CertBlob = *fastly.String(argparser.Content(c.certBlob.Value))
+	}
```

Comment:
The flag is required so there must be a value provided, so we'll just pass the `string` directly to `argparser.Content()`

```suggestion
	input.CertBlob = *fastly.String(argparser.Content(c.certBlob))
```